scripts/update-dependencies.ps1

```diff
@@ -32,9 +32,15 @@ $env:PATH = "$env:DOTNET_INSTALL_DIR;$env:PATH"
 
 $appPath = "$PSScriptRoot\update-dependencies"
 
-# Restore the build scripts
-Write-Host "Restoring Build Script projects..."
-pushd $PSScriptRoot
+# Restore the build_projects
+Write-Host "Restoring Microsoft.DotNet.Cli.Build.Framework..."
+pushd $PSScriptRoot\..\build_projects\Microsoft.DotNet.Cli.Build.Framework
+dotnet restore --infer-runtimes
```

Comment:
Good point.  At one time I had all the RIDs in a "runtimes" section, but that must have never made it into the repo.

I'll add the RIDs and remove --infer-runtimes.